lib/shared/src/lexicalEditor/editorState.test.ts

```diff
@@ -50,16 +58,16 @@ describe('textContentFromSerializedLexicalNode', () => {
                 wrapMention
             )
         ).toBe(
-            'Your task is to generate a suit of multiple unit tests for the functions defined inside the <<file1.go>> file.\n\nUse the <<mention the testing framework>> framework to generate the unit tests. Follow the example tests from the <<mention an example test file>> test file. Include unit tests for the following cases: <<list test cases>>.\n\nEnsure that the unit tests cover all the edge cases and validate the expected functionality of the functions'
+            'Your task is to generate a suit of multiple unit tests for the functions defined inside the <<a/b/file1.go>> file.\n\nUse the <<mention the testing framework>> framework to generate the unit tests. Follow the example tests from the <<mention an example test file>> test file. Include unit tests for the following cases: <<list test cases>>.\n\nEnsure that the unit tests cover all the edge cases and validate the expected functionality of the functions'
         )
     })
 })
 
 describe('editorStateFromPromptString', () => {
     test('converts to rich mentions', async () => {
         const input = ps`What are @${PromptString.fromDisplayPath(
-            URI.file('foo.go')
-        )}:3-5 and @${PromptString.fromDisplayPath(URI.file('bar.go'))} about?`
+            testFileUri('foo.go')
```

Comment:
```suggestion
            displayPath(testFileUri('/foo.go'))
```
if it's fromDisplayPath?